src/test/java/net/notfab/lindsey/core/commands/economy/BlackjackTest.java

```diff
@@ -0,0 +1,44 @@
+package net.notfab.lindsey.core.commands.economy;
+
+import net.notfab.lindsey.core.framework.command.CommandDescriptor;
+import net.notfab.lindsey.core.framework.command.Modules;
+import net.notfab.lindsey.core.framework.command.help.HelpArticle;
+import net.notfab.lindsey.core.framework.command.help.HelpPage;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class BlackjackTest {
+
+    private Blackjack command;
+
+    @BeforeEach
+    void setUp() {
+        command = mock(Blackjack.class);
+        when(command.getInfo())
+                .thenCallRealMethod();
+        when(command.help(null))
+                .thenCallRealMethod();
+    }
+
+    @Test
+    void getInfo() {
+        CommandDescriptor info = command.getInfo();
+        assertEquals("blackjack", info.getName(), "Name must be blackjack");
+        assertEquals(Modules.ECONOMY, info.getModule(), "Module must be economy");
```

Comment:
Maybe the module could be FUN? Others games such as Scramble are using FUN as the module.

This also means moving the command to the correct package.